From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: tabby-machine-user-514235 <tabby-machine-user-514235@gmail.com>
Date: Thu, 1 May 2025 00:01:46 -0700
Subject: force use google chrome's user agent data

This patch forces browser instances to use Google Chrome's user agent
and userAgentData

This patch can be removed when webContents.setUserAgent(...) also
enables setting the navigator.userAgentData

diff --git a/components/embedder_support/user_agent_utils.cc b/components/embedder_support/user_agent_utils.cc
index 608bea5cb386438757146a531bbc0293b3eb9775..d6ac51ffa473fd1126dbcec7c1024acb47ebb4ac 100644
--- a/components/embedder_support/user_agent_utils.cc
+++ b/components/embedder_support/user_agent_utils.cc
@@ -166,6 +166,7 @@ const blink::UserAgentBrandList GetUserAgentBrandList(
 #if !BUILDFLAG(CHROMIUM_BRANDING)
   brand = version_info::GetProductName();
 #endif
+  brand = "Google Chrome";
 
   std::string brand_version =
       output_version_type == blink::UserAgentBrandVersionType::kFullVersion
